client/webserver/site/src/js/markets.js

```diff
@@ -621,6 +687,18 @@ export default class MarketsPage extends BasePage {
     this.showVerify()
   }
 
+  /*
+   * handleFeePayment is the handler for the 'feepayment' notification type.
+   * This is used to update the registration status of the currrent exchange.
+   */
+  handleFeePayment (note) {
+    const dexUrl = note.dexurl
+    if (dexUrl !== this.market.dex.url) return
+    // update local dex
+    this.market.dex = app.exchanges[dexUrl]
```

Comment:
Looking around now, I'm noticing that `this.market.dex` is only ever used for the `dex.url`. Maybe we should just set the field as `this.market.dexAddr`, which won't change, so we don't have to update the `dex` object after `fetchUser` is called.

Reply:
hmm. I don't think I follow you on this one. We also use `this.market.dex` to get the `confs` and `confsRequired` and that's the major reason we update the dex value. 
See [here](https://github.com/decred/dcrdex/pull/353/files#diff-12ae7f4ee80365e11e28e25b80759a65R270).